website/docs/r/stream_connection.html.markdown

```diff
@@ -103,6 +104,11 @@ If `type` is of value `Kafka` the following additional arguments are defined:
 * `broker_public_certificate` - A trusted, public x509 certificate for connecting to Kafka over SSL. String value of the certificate must be defined in the attribute.
 * `protocol` - Describes the transport type. Can be either `PLAINTEXT` or `SSL`.
 
+### DBRoleToExecute
```

Comment:
compute attribute in resources go better in a Attributes Reference section instead of the Argument Reference section, e.g:: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/teams#attributes-reference

Reply:
This isn't a computed attribute

Reply:
oh, i thought it was related to:

"db_role_to_execute": schema.SingleNestedAttribute{
			Computed: true,
			Attributes: map[string]schema.Attribute{
				"role": schema.StringAttribute{
					Computed: true,
				},
				"type": schema.StringAttribute{
					Computed: true,
				},
			},
		},